index.bs

```diff
@@ -286,18 +286,29 @@ Navigation response {#navigation-response}
 At [=populating a session history entry=], in step 6 after substep 7 insert the following:
 <ol>
  <li>Let |shouldReloadForCriticalClientHints| be `false`.
- <li>If <var>navigable</var>'s [=has reloaded for Critical-CH=] is `false`:
+ <li>If <var>navigable</var>'s [=time reloaded for Critical-CH=] is 0:
  <ol>
    <li>Let |shouldReloadForCriticalClientHints| be the result of running [$should reload page for critical client hints$] with the [=relevant settings object=] and |response|.
  </ol>
  <li>Run [$create or override the cached client hints set$] with the [=relevant settings object=] and |response| as inputs.
  <li>If |shouldReloadForCriticalClientHints| then:
  <ol>
-   <li>Set <var>navigable</var>'s [=has reloaded for Critical-CH=] to `true`.
+   <li>Set <var>navigable</var>'s [=time reloaded for Critical-CH=] to the [=current high resolution time=].
    <li><a spec=HTML>reload</a> <var>navigable</var>.
  </ol>
+ <li>If <var>navigable</var>'s [=time reloaded for Critical-CH=] is not 0:
+ <ol>
+    <li>Set <var>navigationParams</var>'s [=critical ch restart time=] to be <var>navigable</var>'s [=time reloaded for Critical-CH=].
+ </ol>
 </ol>
 
+At [=navigation params=], append the following:
+<dl>
+<dt><dfn>critical ch restart time</dfn></dt>
```

Comment:
Where do you read this value?

Reply:
Yes! You should be explicit about where in HTML this is read, in this monkey-patch fashion. (In this and that section, pass this param to `create the navigation timing entry` or so. This way I can go to the link and check that the whole algorithm makes sense.

Reply:
I think I'm already being as specific as the other code in this section? I'm adding it to a struct and specifying it'll be used in "creating the navigation timing entry", just like "[navigation timing type](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-nav-timing-type)".

I hear what you're saying about a missing link, but I'm not seeing a point where it's spelled out any more explicitly.